sync_diff_inspector/tests/sync_diff_inspector/checkpoint/run.sh

```
@@ -50,7 +47,15 @@ echo $first_chunk_info | awk -F '=' '{print $3}' >$OUT_DIR/first_chunk_index
 cat $OUT_DIR/first_chunk_index
 # Notice: when chunk is created paralleling, the least chunk may not appear in the first line. so we sort it as before.
 check_contains "${last_chunk_bound}" $OUT_DIR/first_chunk_bound
-check_contains_regex ".*:${bucket_index_right}-.*:0:.*" $OUT_DIR/first_chunk_index
+OLD_IFS="$IFS"
+IFS=":"
+first_chunk_index_array=($(cat $OUT_DIR/first_chunk_index))
```

Shellcheck (0.11.0)

[warning] In POSIX sh, arrays are undefined.

(SC3030)